Header  header
float64   Acc_0_x
float64   Acc_0_y
float64   Acc_0_z
float64   Alpha_0_x
float64   Alpha_0_y
float64   Alpha_0_z
float64   Omega_0_x
float64   Omega_0_y
float64   Omega_0_z
float64   Pos_0_x
float64   Pos_0_y
float64   Pos_0_z
float64   Vel_0_x
float64   Vel_0_y
float64   Vel_0_z
float64   Acc_B_x
float64   Acc_B_y
float64   Acc_B_z
float64   Alpha_B_x
float64   Alpha_B_y
float64   Alpha_B_z
float64   Omega_B_x
float64   Omega_B_y
float64   Omega_B_z
float64   Vel_B_x
float64   Vel_B_y
float64   Vel_B_z
